libraries/AP_OpticalFlow/AP_OpticalFlow.h

```diff
@@ -100,6 +100,9 @@ class AP_OpticalFlow
     // last_update() - returns system time of last sensor update
     uint32_t last_update() const { return _last_update_ms; }
 
+    // get_height() - returns the user-specified height of sensor above ground
+    float get_height() const { return _height; }
```

Comment:
I think we should rename this to "get_height_override"?

Reply:
Fixed